src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/Conversion.cs

```diff
@@ -25,6 +25,13 @@ namespace Microsoft.CodeAnalysis.CSharp
         // in uncommon cases an instance of this class is attached to the conversion.
         private class UncommonData
         {
+            public static readonly UncommonData NoApplicableOperators = new UncommonData(
```

Comment:
It wasn't clear to me if `UncommonData` is actually accessed when the `UserDefinedConversionResultKind` is `NoApplicableOperators`.

Reply:
I'll try this in a followup pr :)